integrations/react/src/components/Qrcode.jsx

```diff
@@ -0,0 +1,111 @@
+/**
+ * Qrcode.jsx
+ * Airwallex Payment Demo - React.  Created by Chao Ding.
+ *
+ * airwallex-payment-elements Qrcode element integration in React.js
+ * Comments with "Example" demonstrate how states can be integrated
+ * with the element, they can be removed.
+ *
+ * Detailed guidance here: https://github.com/airwallex/airwallex-payment-demo/blob/master/docs/qrcode.md
+ */
+
+import React, { useEffect, useState } from 'react';
+// STEP #1: At the start of your file, import airwallex-payment-elements package
+import { loadAirwallex, createElement } from 'airwallex-payment-elements';
+
+// Enter your Payment Intent secret keys here
+// More on getting these secrets: https://www.airwallex.com/docs/api#/Payment_Acceptance/Payment_Intents/Intro
+const intent_id = 'replace-with-your-intent-id';
+const client_secret = 'replace-with-your-client-secret';
+const payment_method = 'replace-with-your-payment-method'; // available methods: 'alipayhk', 'kakaopay', 'wechatpay'
+
+const Qrcode = () => {
+  const [elementShow, setElementShow] = useState(false); // Example: set loading state for element
+  const [errorMessage, setErrorMessage] = useState(false); // Example: set error state
+
+  useEffect(() => {
+    // STEP #2: Initialize Airwallex on mount with the appropriate production environment and other configurations
+    loadAirwallex({
+      env: 'demo', // Can choose other production environments, 'staging | 'demo' | 'prod'
+      origin: window.location.origin, // Setup your event target to receive the browser events message
+    }).then(() => {
+      // STEP #4: Create the qrcode card element
+      const qrcode = createElement('qrcode', {
+        intent: {
+          // Required, must provide intent details to prepare qrcode element for checkout
+          id: intent_id,
+          client_secret,
+        },
+        qrcodeMethod: payment_method,
+      });
+      // STEP #5: Mount the element to the empty container created previously
+      qrcode.mount('qrcode'); // This 'qrcode' id MUST MATCH the id on your empty container created in Step 3
```

Comment:
MUST MATCH why is  all uppercase

Reply:
I think want to make it more important
believe in @josie-ku-awx , she is professional in this.